### ISSUE TITLE
Autocomplete/Standalone search box

### ISSUE THREAD

**Issue by Moramu** [JustFly1984/react-google-maps-api#151]
# Please provide an explanation

## Your Environment

os: windows/

node --version 10.13.0

react version 16.8.6

webpack version "4.33.0"

@babel version 7.4.3

@react-google-maps/api version 1.2.7

I just migrated from old version of react-google-maps, which is no longer supported. 
I successfully implemented Markers and InfoWindow from your package, but when I'm trying to use Autocomplete or Standalone search box I have an issue: 

```
 <LoadScript googleMapsApiKey={""}>     
              <GoogleMap
                  id="some-map"
                  zoom={9}
                  center={{lat: 39.9525839, lng: -75.16522150000003}}
              >
             <Autocomplete
                onLoad={this.onLoad}
               onPlacesChanged={this.onPlaceChanged}
             >
            <input
              type="text"
              placeholder="Customized your placeholder"
            />
          </Autocomplete>
              </GoogleMap>
</LoadScript>     
```

![Capture](https://user-images.githubusercontent.com/33767925/59237826-91a97500-8bc9-11e9-994c-d71d20382318.PNG)

What I'm missing...

**Comment by JustFly1984** [comment]
@Moramu can you please provide a link to minimal example at codesandbox?

**Comment by Moramu** [comment]
Yes, i can, but codesandbox temporary saying "Could not fetch dependencies, please try again in a couple seconds: Something went wrong while packaging the dependency @react-google-maps/api@1.2.7"

**Comment by Moramu** [comment]
Codesandbox, showing me error while fetching script so i use another one.... 
https://stackblitz.com/edit/react-mrzxru

**Comment by JustFly1984** [comment]
https://stackblitz.com/edit/react-cmo3u7?file=index.js

You have to add an array of libraries including "places" string like I did in the fork of your stackblitz.

I will add that to example.

**Comment by JustFly1984** [comment]
I have published 1.3.0, fixed an error messages by invariant
